core/src/shred_fetch_stage.rs

```diff
@@ -433,6 +413,7 @@ pub(crate) fn receive_quic_datagrams(
 
 // Returns true if the feature is effective for the shred slot.
 #[must_use]
+#[allow(dead_code)]
```

Comment:
Need to keep feature checking machinery for https://github.com/anza-xyz/agave/pull/2149.
This will be removed in https://github.com/anza-xyz/agave/pull/2149.